include/llvm/Target/TargetOptions.h

```diff
@@ -217,9 +217,9 @@ namespace llvm {
     /// This flag enables InterProcedural Register Allocation (IPRA).
     unsigned EnableIPRA : 1;
 
-    /// PositionIndependentPages- This flag indicates that PC-relative
+    /// PositionIndependentPages - This flag indicates that PC-relative
     /// references outside the current function are invalid and all global and
-    /// inter-function references must go through the Page Linking Table (PGLT)
+    /// inter-function references must go through the Page Offset Table (POT)
     unsigned PositionIndependentPages : 1;
```

Comment:
Should this just be "Pagerando"? I think that makes sense because we reference pagerando in other places. Maybe `EnablePagerando`?